### DISCUSSION TITLE
Even though I am approved , my github education benefits are still not available 3 days have already passed.

### DISCUSSION THREAD

**Discussion by x420520** [community/community#142751] [GitHub Education]
### Hi everyone,

Hi everyone,
Hi everyone,
I recently got approved for Github Education but I have not gotten my access to the benefits and I am worried because it has been a few days. Is it possible to know when I can get it . Thank po !

**Comment by Inmoresentum** [comment]
Please wait patiently! It may take upto a week and you should receive email regarding your updates and stuff. I hope this helps.

**Comment by davevad93** [comment]
Hi @x420520 ,

Don't worry, the waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @x420520 was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
